src/hooks/useNetworkInfo.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  ASTAR_EVM_NETWORK_IDX,
+  ASTAR_NATIVE_TOKEN,
+  ASTAR_NETWORK_IDX,
+  getProviderIndex,
+  providerEndpoints,
+} from 'src/config/chainEndpoints';
+import { useStore } from 'src/store';
+import { computed } from 'vue';
+
+export function useNetworkInfo() {
+  const store = useStore();
+
+  const isMainnet = computed<boolean>(() => {
+    const chainInfo = store.getters['general/chainInfo'];
+    const network = chainInfo ? chainInfo.chain : '';
+    const isTestnet = network === 'Development' || network === 'Shibuya Testnet';
```

Comment:
maybe we should define this network name as external constant value

Reply:
I have fixed in this commit https://github.com/AstarNetwork/astar-apps/pull/469/commits/ed79f2ae9bf5c316d52f3683f83024e8673eb5ca